index.html

```diff
@@ -0,0 +1,98 @@
+          </div>
+          <div class="phone">phone: +(0) 1 2345 555</div>
+          <div class="email">email: john@doe.com</div>
+        </div>
+      </div>
+    </div>
+  </header>
+  <main class="content">
+    <div class="info">
+      <div class="about">
+        ABOUT ME
+      </div>
+      <div class="about-text">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed ut tortor eu lectus consectetur mattis. Cras a turpis ut leo auctor auctor. Etiam nulla erat, gravida vel elit in, scelerisque laoreet erat. Phasellus rutrum velit dolor, quis porta tortor aliquam non. Nunc in risus faucibus, ultrices tortor sed, elementum augue.
+      </div>
+      <div class="about-additional">
+        Proin dignissim, eros ac consectetur venenatis, lorem nibh iaculis ligula, id suscipit orci est auctor nibh. Cras placerat non ligula eget ultrices. Integer sed ultrices nunc. Integer quis luctus massa.
+      </div>
+      <div class="skills">SKILLS</div>
+      <div class="skills-text">
+        <div class="skills-container">
+          <div class="graphic">Graphic Design</div>
+          <div class="progress-bar-back">
+            <div class="progress-bar-front90"></div>
+          </div>
+        </div>
+        <div class="skills-container">
+          <div class="developer">Web Develop</div>
+          <div class="progress-bar-back">
+            <div class="progress-bar-front90"></div>
+          </div>
+        </div>
+        <div class="skills-container">
+          <div class="lorem">Lorem Ipsum</div>
+          <div class="progress-bar-back">
+            <div class="progress-bar-front75"></div>
+          </div>
+        </div>
+        <div class="skills-container">
+          <div class="dolor">Dolor sit amet</div>
+          <div class="progress-bar-back">
+            <div class="progress-bar-front80"></div>
+          </div>
+        </div>
+        <div class="skills-container">
+          <div class="elit">Consectetur elit</div>
+          <div class="progress-bar-back">
+            <div class="progress-bar-front90"></div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="qualifications">
+      <div class="experience">EXPERIENCE</div>
+      <div class="job-position">Sales Force Team Leader (2006 - NOW)</div>
+      <div class="experience-text">Sed et tincidunt velit, a condimentum orci. Sed quis pellentesque nisl. Maecenas et ullamcorper nulla, ut vehicula augue. Nam at ex lacinia, accumsan nunc in, ultricies nunc.
+      </div>
+      <div class="job-position">Sales Manager (2003 - 2006)</div>
+      <div class="experience-text">Mauris porttitor mauris a nunc elementum, ut consequat libero tincidunt. Curabitur egestas sit amet tellus nec blandit.
+      </div>
+      <div class="job-position">Sales Person (1999 - 2003)</div>
+      <div class="experience-text"> Duis iaculis sem id massa bibendum, et auctor nibh tincidunt. Nulla velit metus, porta ac nulla ac, rhoncus pulvinar eros.

```

Comment:
this space is not needed
```suggestion
      <div class="experience-text">Duis iaculis sem id massa bibendum, et auctor nibh tincidunt. Nulla velit metus, porta ac nulla ac, rhoncus pulvinar eros.

```

Reply:
Done